_data/tools.json

```
@@ -1,4 +1,13 @@
 [
+   {
+      "title": "Static Reviewer",
+      "url": "https://www.securityreviewer.net/",
+      "owner": "Security Reviewer",
+      "license": "Commercial",
+      "platforms": "Windows and Linux; on-Premises and in Cloud; Desktop, CLI and CI/CD & IDE plugin integration",
+      "note": "Static Reviewer executes code checks according most relevant Secure Coding Standards for 40+ programming languages, using built-in design 1000+ validation rules.",
```

```suggestion
      "note": "Static Reviewer executes code checks according to the most relevant Secure Coding Standards for 40+ programming languages, using built-in design 1000+ validation rules.",
```

Seems like there’s a word missing after “design” as well, but I have no idea what you were going for… 🤷‍♂️

---

"...using built-in 1000+ validation rules." the word "design" is a copy/paste error. Thx.